classify_emails.py

```
@@ -310,13 +315,13 @@ def classify_emails(checkpoint: dict) -> int:
     return classified
 
 
-def run_classification():
+def run_classification(reset_registry: bool = False):
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)